### PR TITLE
TNO-679: Allow admins to edit user information

### PR DESCRIPTION
**Pull request by jdtoombs**
Everything was setup already, just needed to check if the user was an admin or not

### CODE REVIEW

File: app/editor/src/features/admin/users/UserForm.tsx [hunk 1]
```diff
@@ -40,7 +41,7 @@ export const UserForm: React.FC = () => {
     lookups.roles.map((r) => new OptionItem(r.name, r.id)),
   );
 
-  const isLinkedToKeycloak = user.key !== '00000000-0000-0000-0000-000000000000';
+  const isAdminUser = isAdmin(user);
```
Comment (Fosol) [inline app/editor/src/features/admin/users/UserForm.tsx:44]: You don't need this.  The admin is the only user who should be able to get to this form.

File: app/editor/src/features/admin/users/utils/isAdmin.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { IUserModel } from 'hooks';
+
+/**
+ * Check whether the user is an admin or not.
+ * @param user the user to check
+ * @returns  true if the user is an admin, false otherwise
+ */
+export const isAdmin = (user: IUserModel) => {
```
Comment (Fosol) [inline app/editor/src/features/admin/users/utils/isAdmin.ts:8]: Not required

File: app/editor/src/features/admin/users/UserForm.tsx [hunk 3]
```diff
@@ -40,7 +41,7 @@ export const UserForm: React.FC = () => {
     lookups.roles.map((r) => new OptionItem(r.name, r.id)),
   );
 
-  const isLinkedToKeycloak = user.key !== '00000000-0000-0000-0000-000000000000';
```
Comment (Fosol) [inline app/editor/src/features/admin/users/UserForm.tsx:43]: This logic was to protect users that existed in Keycloak.  The reality is that in the future we won't be able to edit users in keycloak anyway.  So I'm wondering if there is any value in making this change.  For now you can simply comment this line of code out, as something similar will need to be added back.

**Review by Fosol — CHANGES_REQUESTED** [review]

File: app/editor/src/features/admin/users/UserForm.tsx (hunk 3 above)
Comment (jdtoombs) [reply app/editor/src/features/admin/users/UserForm.tsx:43]: Sounds good!

File: app/editor/src/features/admin/users/UserForm.tsx (hunk 1 above)
Comment (jdtoombs) [reply app/editor/src/features/admin/users/UserForm.tsx:44]: Figured that may be the case... will get rid of

**Pull request closed without merging**